### PR TITLE
Audit Item N08

### LINKED ISSUE #39: Resolve audit item N08

### PR DESCRIPTION
**Pull request by apbendi**
closes #39

### CODE REVIEW

File: contracts/Org.sol [hunk 1]
```diff
@@ -13,7 +13,7 @@ import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
  * @notice Org is a contract that serves as a smart wallet for US nonprofit
  * organizations. It holds the organization's federal Tax ID number as taxID, 
  * and allows for an address to submit a Claim struct to the contract whereby 
- * the organization can direct recieved grant awards from Endaoment Funds.
+ * the organization can directly recieved grant awards from Endaoment Funds.
```
Comment (CantelopePeel) [inline contracts/Org.sol]: s/directly recieved/directly receive/

**Review by CantelopePeel — CHANGES_REQUESTED** [review]
One small item

File: contracts/Org.sol (hunk 1 above)
Comment (apbendi) [reply contracts/Org.sol]: good catch!

**Review by CantelopePeel — APPROVED** [review]
LGTM

**Pull request merged**